deno_runtime/README.md

```diff
@@ -0,0 +1,10 @@
+# TODO(bartlomieju):
+
+# `deno_runtime` crate
+
+[![crates](https://img.shields.io/crates/v/deno.svg)](https://crates.io/crates/deno)
+[![docs](https://docs.rs/deno/badge.svg)](https://docs.rs/deno)
+
+This provides the actual deno executable and the user-facing APIs.
+
+The deno crate uses the deno_core to provide the executable.
```

Comment:
The links are wrong.

It would be v useful to have a good description of the purpose and scope of this crate here.

Reply:
I've updated the README, but don't really feel like writing prose today 😅